app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,40 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
-  border-collapse: collapse
-  thead
-    border: solid 1px $maincolor
-  td, th
+  tbody, tr, thead
+    display: contents
+  th
+    border-bottom: solid 1px $maincolor
+    background-color: $softbgcolor
     padding: 2px 4px
   th.manage
     padding: 2px 0
-  tr.tableheader
-    background-color: $softbgcolor
   tr.poll
     td
+      padding: 3px 4px
       cursor: default
-    &:hover
-      background-color: $softbgcolor
+    &:hover td
+     background-color: $softbgcolor
+    td[colspan="2"]
+      grid-column: span 2
+    td.title
+      align-self: center
+      text-overflow: ellipsis
+      overflow: hidden
+      white-space: nowrap
     td.round, td.deadline, td.terminated
+      align-self: center
       text-align: center
+      white-space: nowrap
     .last-line
+      align-self: center
```

Comment:
> The alignment is better, thanks. But the issue with the background color of td.manage is still there. The issue is on line without the dustbin (but in table where some lines have the dustbin). There are two symptoms:
> 
> * If you leave the mouse over the empty td.manage cell, the background of the line stay white.
> * If you leave the mouse somewhere else on the line (for instance over the title), the background of td.manage doesn't change (the background change on all others cells).

Okay Thanks! With the setting "align-self: center" for every separated td, the browser Chrome and Firefox can display the background right on my screen, but the "align-items: center" for the general td makes the mistake. Like that: 
![image](https://user-images.githubusercontent.com/72395200/121710579-0b86d880-cada-11eb-86c1-df3e2c7ff4c9.png)

Reply:
The [doc](https://www.w3.org/TR/css-grid-1/#column-align) says:
> by using the align-self property on the grid item or align-items property on the grid container

td elements are not grid containers.

Reply:
Sorry, my bad, it's friday... You're right: empty td.manage with center align have height = 0, hence no background. Still, I find it's better to say "all the td have center align, except td.manage" instead of "td.title, td.round, and td.action have center align".

Reply:
Okay!! So I just need to correct the space in the indentation for the "&:hover", is that alright ~?